uber/config.py

```diff
@@ -904,6 +904,8 @@ class AWSSecretFetcher:
 
     def __init__(self):
         import boto3
+
+        region_name = "us-east-2"
```

Comment:
hard coded for expediency, but we should fix it before we merge

Reply:
This is now fixed!